### DISCUSSION TITLE
Will I have to retrain with whole dataset if i want to add some new images with same classes

### DISCUSSION THREAD

**Discussion by dhunjoshi** [ultralytics/ultralytics#716] [General]
I have train a model previously with large dataset . But in Some Images it does not give correct result. I want to retrain with previous weight . Now I want to know can i retrain with only new images or i have to retrain with whole dataset including new images?

**Comment by Laughing-q** [comment]
@dhunjoshi the whole dataset including new images.

**Reply by dhunjoshi** [reply]
Thanks.

**Reply by glenn-jocher** [reply]
@dhunjoshi you're welcome! If you have any more questions or need further assistance, feel free to ask. Happy training! 😊🚀
